### PR TITLE
Changes PT-BR

### PR DESCRIPTION
**Pull request by bitcoinuser**

### CODE REVIEW

File: phoenix-legacy/src/main/res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -585,7 +585,7 @@
   <!-- //////////////// display preferences //////////////// -->
 
   <string name="prefs_display_title">Opções de exibição</string>
-  <string name="prefs_display_coin_label">Unidade de Bitcoin</string>
+  <string name="prefs_display_coin_label">Unidade de bitcoin</string>
   <string name="prefs_display_fiat_label">Moeda estatal</string>
```
Comment (bitcoinuser) [inline phoenix-legacy/src/main/res/values-pt-rBR/strings.xml:589]:
```suggestion
  <string name="prefs_display_fiat_label">Moeda fiduciária</string>
```

File: phoenix-legacy/src/main/res/values-pt-rBR/strings.xml [hunk 2]
```diff
@@ -741,7 +741,7 @@
 
   <string name="paymentmode_onchain">Pagar usando a rede Bitcoin</string>
   <string name="paymentmode_onchain_desc">Transações na rede Bitcoin geralmente são mais lentas e mais caras</string>
-  <string name="paymentmode_lightning">Pagar usando a rede Lightning</string>
+  <string name="paymentmode_lightning">Pagar usando a rede relâmpago</string>
```
Comment (bitcoinuser) [inline phoenix-legacy/src/main/res/values-pt-rBR/strings.xml:744]:
```suggestion
  <string name="paymentmode_lightning">Pagar usando a rede Relâmpago</string>
```

File: phoenix-legacy/src/main/res/values-pt-rBR/strings.xml [hunk 3]
```diff
@@ -647,7 +647,7 @@
   <string name="about_title">Sobre</string>
   <string name="about_version">Versão: %1$s (%2$d)</string>
   <string name="about_general"><![CDATA[
-  A Phoenix é uma carteira Bitcoin que usa a rede Lightning para enviar e receber pagamentos.
+  A Phoenix é uma carteira bitcoin que usa a rede Relâmpago para enviar e receber pagamentos.
```
Comment (bitcoinuser) [inline phoenix-legacy/src/main/res/values-pt-rBR/strings.xml:650]:
```suggestion
  A Phoenix é uma carteira Bitcoin que usa a rede Relâmpago para enviar e receber pagamentos.
```

File: phoenix-legacy/src/main/res/values-pt-rBR/strings.xml [hunk 4]
```diff
@@ -694,7 +694,7 @@
   <string name="paymentsettings_paytoopen_fees_desc">%1$s%% (%2$s mínimo)</string>
   <string name="paymentsettings_paytoopen_fees_dialog_title">Abertura de canais</string>
   <string name="paymentsettings_paytoopen_fees_dialog_message"><![CDATA[
-    Esta taxa se aplica quando você recebe um pagamento pela rede Lightning e um novo canal precisa ser aberto.
+    Esta taxa se aplica quando você recebe um pagamento pela rede relâmpago e um novo canal precisa ser aberto.
```
Comment (bitcoinuser) [inline phoenix-legacy/src/main/res/values-pt-rBR/strings.xml:697]:
```suggestion
    Esta taxa se aplica quando você recebe um pagamento pela rede Relâmpago e um novo canal precisa ser aberto.
```

File: phoenix-legacy/src/main/res/values-pt-rBR/strings.xml [hunk 5]
```diff
@@ -679,7 +679,7 @@
   <string name="paymentsettings_defaultdesc_dialog_description">As faturas que você gerar irão conter esta descrição. Você pode substituí-las caso a caso.</string>
   <string name="paymentsettings_defaultdesc_dialog_hint">Insira uma descrição</string>
 
-  <string name="paymentsettings_trampoline_fees_title">Taxa máxima para pagamentos pela rede Lightning</string>
+  <string name="paymentsettings_trampoline_fees_title">Taxa máxima para pagamentos pela rede relâmpago</string>
```
Comment (bitcoinuser) [inline phoenix-legacy/src/main/res/values-pt-rBR/strings.xml:682]:
```suggestion
  <string name="paymentsettings_trampoline_fees_title">Taxa máxima para pagamentos pela rede Relâmpago</string>
```

**Review by dpad85 — APPROVED** [review]
Looks good, thanks! Interesting that you also translated `Lightning` to Portuguese.

**Pull request merged**